codice/ddf
# DDF-1970: As an Integrator, I want information on Migratables.

#### What does this PR do?
Extends `platform-migration` to look-up and return migratables that also implement describable. Warnings are logged when migratables are found that do not conform to this criteria.  

#### Who is reviewing it?
@lessarderic @garrettfreibott @oconnormi @figliold 

#### How should this be tested?
Full build - make sure DDF starts and that `migration:export` works. No need to ingest anything. 

#### Any background context you want to provide?
We are exposing functionality for later consumption in the CLI and GUI. 

#### What are the relevant tickets?
DDF-1970

#### Screenshots (if appropriate)

#### Checklist:
